features/validators.feature

```diff
@@ -5,11 +5,7 @@ Feature: Data validators and output format
   Scenario: Validators output format
```

Comment:
I assume this file is still WIP?

Reply:
Yes. I wanted to discuss whether we need this feature, as it's not a Gavel's feature, but rather test suites for Gavel internals. These belong more to unit tests of Gavel than to gavel-spec. 

Instead, we should write feature files for separate fields (`body.feature`, `statusCode.feature`) to express how the validation of those fields must go.

What do you think?

Reply:
Also I think this block is too technical to be in a feature description:

```gherkin
Given you call:
    """
    const gavel = require('gavel');
    """
```

It also couples the feature descriptions with the way the library is exported.

Reply:
I agree. The "feature" here shouldn't be validators, but e.g. "body validation".

Reply:
Separate fields validations features have been added.